### PR TITLE
Refactor/exchange state

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | none
| License          | BSD 3-Clause

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#14419.0 ✘  
* 
  * Build ✘

**Comment by antleblanc** [comment]
@marie-j Could you please resolve existing conflicts? In order to be able to merge it.

Thank you!

**Comment by marie-j** [comment]
> @marie-j Could you please resolve existing conflicts? In order to be able to merge it.
> 
> Thank you!

@antleblanc Done !

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import ExchangeModel from '@ovh-ux/manager-exchange/src/Exchange.class'; // eslint-disable-line import/extensions
+import { Exchange as ExchangeModel } from '@ovh-ux/manager-exchange'; // eslint-disable-line import/extensions
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js:1]: I think the ESLint comment could be removed:
```suggestion
import { Exchange as ExchangeModel } from '@ovh-ux/manager-exchange';
```

File: packages/manager/modules/exchange/src/shared/shared.html [hunk 2]
```diff
@@ -227,8 +224,8 @@
         </div>
     </div>
 
-    <div
+    <!-- <div
```
Comment (antleblanc) [inline packages/manager/modules/exchange/src/shared/shared.html:227]: **suggestion:**  :fire:

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js:1]: I think it was done due to the previous import but I forgot to remove it

**Comment by marie-j** [comment]
I've fixed the two (nitpicking) comments

**Pull request merged**